src/main.ts

```diff
@@ -3,48 +3,43 @@ import 'core-js/stable' // to polyfill ECMAScript features
 import 'regenerator-runtime/runtime' // to use transpiled generator functions
 
 // Vue Libraries
-import Vue from 'vue'
-import Vuetify from 'vuetify'
+import { createApp } from 'vue'
 import Vuelidate from 'vuelidate'
 import { getVueRouter } from '@/router'
-import { getPiniaStore, getVuexStore } from '@/store'
+import { getPiniaStore } from '@/store'
 import Affix from 'vue-affix'
-import Vue2Filters from 'vue2-filters' // needed by SbcFeeSummary
```

Comment:
Deprecated, they don't exist anymore in Vue 3. Reference: https://v3-migration.vuejs.org/breaking-changes/filters.html

Reply:
bcgov/entity#13653